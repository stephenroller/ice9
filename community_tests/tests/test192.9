# more string tests (first 4 pass - last fails """)			fail

"#";
'#';

"'";
'"'

""";