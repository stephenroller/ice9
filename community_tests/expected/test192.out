line 9: illegal character (")
